test/Multichain.js

```diff
@@ -47,7 +47,7 @@ describe('Multichain', () => {
                 })
             })
     
-            it('should return empty list when key not found', function(done) {
+            it('Returns an empty list when key not found', function(done) {
```

Comment:
returns an empty list with no error when key not found

Reply:
done